python/src/com/google/idea/blaze/python/resolve/provider/BazelPyImportResolverStrategy.java

```diff
@@ -51,15 +51,6 @@ protected QualifiedName toImportString(ArtifactLocation source) {
     if (source.isGenerated() || !source.getRelativePath().endsWith(".py")) {
       return null;
     }
-    QualifiedName name = fromRelativePath(source.getRelativePath());
```

Comment:
I have observed that in this situation, the associated `py_library` is already coming through with an `imports` and so this PR will handle this situation and there should no longer be a need to perform this work-around.